.gitignore

```diff
@@ -50,3 +50,31 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+*.js
+*.html
+*.png
+*.css
+build/CMakeFiles/stm32_spinner_project.dir/link.txt
+build/Debug/stm32_spinner_project.bin
+build/.cmake/api/v1/reply/codemodel-v2-266db49fb6621a6af4fd.json
```

Comment:
you  can specify build/* instead these files

Reply:
done